test/lru_cache_test.rb

```diff
@@ -0,0 +1,63 @@
+require 'test_helper'
+
+class TestLRUCache < Minitest::Test
+  def setup
+    @lru_cache = LRUCache.new(3)
+  end
+
+  def test_set_item
+    @lru_cache.set('key', 'value')
+    assert @lru_cache.table['key'].value == 'value'
```

Comment:
I tend to dislike tests that assert on implementation details like this.

Don't we really just want to test that a key value that was set is returned by get?